Sources/BraintreePayPal/Public/BraintreePayPal/BTPayPalCheckoutRequest.h

```diff
@@ -65,6 +65,11 @@ typedef NS_ENUM(NSInteger, BTPayPalRequestIntent) {
  */
 @property (nonatomic) BOOL offerPayLater;
 
+/**
+ Optional: If set to true, this enables the Checkout with Vault flow, where the customer will be prompted to consent to a billing agreement during checkout.
+ */
+@property (nonatomic) BOOL requestBillingAgreement;
```

Comment:
There was some discussion in the previous PR about whether to call this something else, like `checkoutWithVault`. I went with `requestBillingAgreement` since that's what we use on JS, but I'm curious what other people think makes the most sense.

Reply:
I like that it lines up with JS. An alternative could be `checkoutWithVaultEnabled`, but SDK parity feels like the best call.